### ISSUE TITLE
New Scheduler change causes repeat errors

### ISSUE THREAD

**Issue by thebgpikester** [FlightControl-Master/MOOSE#1195]
https://github.com/FlightControl-Master/MOOSE/commit/14b35cb0692c41f092dc475c4800a3697d431e99#diff-5885dad176ad704119ad72de04a9cc75R172

causes:

```
2019-09-13 14:29:30.585 INFO    SCRIPTING: Error in timer function: [string "C:\Users\KALLEN~1\AppData\Local\Temp\K-0-1/~mis000021CB.lua"]:5526: attempt to concatenate upvalue 'Source' (a nil value)
2019-09-13 14:29:32.759 WARNING LOG: 237 duplicate message(s) skipped.
```

Apparently on dedicated server but not normal server. Do not know why. Reported multiple times by multiple people.

**Comment by Delta-99** [comment]
See my comment on Pikes comment. A line setting Source to "" seems to have been deleted. Most likely by accident. Probably a super easy fix.

**Comment by funkyfranky** [comment]
Fixed in #1197
